Transformer/README.md

```diff
@@ -5,26 +5,37 @@ A transformer is responsible to semantically transform an incoming JSON object i
+A transformer expects a JSON object which represents the application's proprietary data model as an input. It is possible to automatically pull JSON schemas via the OIH API which represent the structure of the Open Integration Hub master data model. Afterwards it transforms the incoming JSON object into another JSON object via a transformation language (_We suggest JSONata for the transformation_). The resulting JSON object represents the structure of the Open Integration Hub master data model. The transformer's output is then send to the Open Integration Hub and is validated against a deposited JSON schema.
 
 ## Transformation Language
-A transformation language is needed to transform one object into another. For the semantic transformation that is performed by the transformer, we suggest JSONata as it is built to transform one JSON object into another.
+One way of transforming the JSON objects is the usage of a transformation language. As already mentioned our suggested transformation language is JSONata as it is especially built to transform one JSON object into another.
 
-**JSONata:**
-- Is a Lightweight query and transformation language for JSON data
-- Helps to extract specific meaningful data from JSON data using a simple logic
-- Supports more than 20 operators
-- Includes numerous functions such as string- and numeric-functions
-
-For further information of JSONata please visit the [official JSONata documentation](http://jsonata.org/).
+For detailed information on transformation language and JSONata as well as a general example please have a look at [TransformationLanguage](/TransformationLanguage.md)
 
 ## Transformer Evolution
-Sometimes there is a need to change the existing data model to adjust to different requirements. In this case, a transformer needs to be adjusted/updated in order to be compatible with the newest version of the data model (Backward compatibility is supported, therefore an existing transformer can still be used with previous versions of the model).
+Sometimes there is a need to change the existing data model to adjust to different requirements. In this case, a transformer needs to be adjusted/updated in order to be compatible with the newest version of the data model.
+
+Some changes do not affect existing mappings/transformations and transformers can still be used (although they are not reffering to the newest model version). Thus, backward compatibility is automatically given. According to the [OData specification](http://docs.oasis-open.org/odata/odata/v4.0/errata03/os/complete/part1-protocol/odata-v4.0-errata03-os-part1-protocol-complete.html#_Toc453752210) (shortened/modified list) the following changes do not require a change within a transformer:
```

Comment:
`referring`

Reply:
Was your question answered during the call?

Reply:
I think so.